test/rules/always_put_control_body_on_new_line.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `pub run test -N always_put_control_body_on_new_line`
+
+testIfElse() {
```

Comment:
I agree.

I see this lint as a workaround until a flutter formatter lands. However once a formatter will be used you will face the same kind of issue/concern : the same rule for all the code.

But I don't think there will be any `//ignore`. When I run this lint on the Flutter codebase I have no lint anymore (except for a `while (...) { }`) since https://github.com/flutter/flutter/pull/9177

Obviously this lint could be removed when the flutter formatter will land.